client/src/main/com/sinch/sdk/domains/verification/api/v1/VerificationService.java

```diff
@@ -1,12 +1,51 @@
 package com.sinch.sdk.domains.verification.api.v1;
 
+/**
+ * Verification Domain Level Service
+ *
+ * @see <a
+ *     href="https://developers.sinch.com/docs/verification">https://developers.sinch.com/docs/verification</a>
+ * @since 1.1
+ */
 public interface VerificationService {
 
+  /**
+   * Verifications Start Service instance
+   *
+   * @return service instance for project
+   * @see <a
+   *     href="https://developers.sinch.com/docs/verification/api-reference/verification/tag/Verifications">Documentation</a>
```

Comment:
Is it intended to display "Documentation" only?

Reply:
It could be changed to highlight the dedicated documentation section pointed by the link